src/services/etherspot.js

```diff
@@ -73,11 +81,25 @@ class EtherspotService {
       ? EnvNames.MainNets
       : EnvNames.TestNets;
 
-    this.sdk = new EtherspotSdk(privateKey, { env: envName, networkName });
-
-    await this.sdk.computeContractAccount({ sync: true }).catch((error) => {
-      reportErrorLog('EtherspotService init computeContractAccount failed', { error });
+    /**
+     * Cycle through the supported networks and build an
+     * array of instantiated instances
+     */
+    this.supportedNetworks.forEach((currentNetworkName) => {
+      // Instantiate
+      this.instances[currentNetworkName] = new EtherspotSdk(privateKey, { env: envName, currentNetworkName });
+
+      // Build up an array of computeContractAccount's to be executed later
```

Comment:
I would suggest clarifying the comment a bit
```suggestion
      // Schedule exection of computeContractAccount's
```

Reply:
Done ✅